js/csv.js

```diff
@@ -0,0 +1,195 @@
+/**
+ * Converts a JSON object to a valid CSV string file.
+ *
+ * @param {Object} object The object to convert.
+ * @returns {String} The converted string.
+ */
+export const objectToCsv = object => {
+    let csv = "";
+    if (Object.keys(object) === 0) return csv;
+
+    const fields = Object.keys(object[0]);

```

Comment:
Why `object[0]`?

Reply:
Since in theory all of the objects share the number of fields (only that some of them are null), the first object contains all the information that is necessary to get the fields information. We use the first simply because it's the easiest to access while making sure that we're not accessing out of bounds information.